skyfield/functions.py

```diff
@@ -22,6 +22,23 @@ def length_of(xyz):
 
     """
     return sqrt((xyz * xyz).sum(axis=0))
+    
+def angle_between(u_vec, v_vec):
```

Comment:
OK, I'll change this to the acos formula used by ``separation_from()``.

Reply:
I moved the math used by separation_from into a function in ``functions.py`` called ``angle_between()`` so I could reuse it in ``elementslib.py``. I also renamed my version from ``angle_between()`` to ``angle_between2()``.

``elementslib.py`` continues to use ``angle_between2()`` because using ``angle_between()`` raises errors, but I don't yet know why.